data/crac-file/crac-io-json/src/test/java/com/farao_community/farao/data/crac_io_json/CracImportExportTest.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 20, RTE (http://www.rte-france.com)
```

Comment:
Computers didn't exist in the year 20: they appeared several centuries later...

Reply:
hum...